cli/index.js

```diff
@@ -81,4 +431,425 @@ y.command({
+            demandOption: true,
+            type: 'string'
+        },
+       picture: {
+           describe: 'A profile picture',
+            demandOption: true,
+            type: 'string'
+        }
+        // TODO: prompt to confirm the password
+    },
+    handler(argv) {
+        guest(argv)
+    }
+})
+
+// Logout user
+y.scriptName("connectme")
+y.command({
+    command: 'logout',
+    describe: 'Logout current account ',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+        userID: {
+            describe: 'An user ID',
+            demandOption: true,
+            type: 'string'
+        },
+        
+    },
+    handler(argv) {
+        logOut(argv)
+    }
+})
+
+// recover the account
+y.scriptName("connectme")
+y.command({
+    command: 'recover',
+    describe: 'recover the current account ',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+        userID: {
+            describe: 'An user ID',
+            demandOption: true,
+            type: 'string'
+        },
+        email: {
+            describe: 'An user email',
+            demandOption: true,
+            type: 'string'
+        },
+        newPassword: {
+            describe: 'The new password of the account',
+            demandOption: true,
+            type: 'string'
+        },
+        confirmPassword: {
+            describe: 'confirm the current password',
+            demandOption: true,
+            type: 'string'
+        },  
+    },
+    handler(argv) {
+        recover(argv)
+    }
+})
+
+// delete an account 
+y.scriptName("connectme")
+y.command({
+    command: 'delete',
+    describe: 'This can only be done by the logged in user',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+    },
+    handler(argv) {
+        deleteuser(argv)
+    }
+})
+
+// get user by username
+y.scriptName("connectme")
+y.command({
+    command: 'get-user',
+    describe: 'get an user by username',
+    builder: {
+        username: {
+            describe: 'Username',
+            demandOption: true,
+            type: 'string'
+        },
+    },
+    handler(argv) {
+        getuser(argv)
+    }
+})
+
+// update account
+y.scriptName("connectme")
+y.command({
+    command: 'update-user',
+    describe: 'update & existen account',
+    builder: {
+        userId: {
+            describe: 'and userID',
+            demandOption: true,
+            type: 'string'
+        },
+        password: {
+            describe: 'password of the account',
+            demandOption: true,
+            type: 'string'
+        },
+        confirmPassword: {
+            describe: 'confirm the password',
+            demandOption: true,
+            type: 'string'
+        },

```

Comment:
it doesn't make sense to make the user update his password as part of this update call, because we already have the `recover` endpoint to do exactly this

Reply:
done